src/routes/safe/store/actions/createTransaction.js

```diff
@@ -78,7 +78,7 @@ const createTransaction = ({
   const from = userAccountSelector(state)
   const safeInstance = await getGnosisSafeInstanceAt(safeAddress)
   const threshold = await safeInstance.getThreshold()
-  const nonce = txNonce || await getLastPendingTxNonce(safeAddress)
+  const nonce = [undefined, null, ''].includes(txNonce) ? await getLastPendingTxNonce(safeAddress) : txNonce
```

Comment:
can we maybe use `Number.isInteger`? This check looks too fancy now :D 
or `txNonce == null` (notice two equal signs and not 3)

Reply:
Yes, totally!!